vars/buildRpm.groovy

```diff
@@ -1,5 +1,9 @@
 /* groovylint-disable DuplicateStringLiteral, VariableName */
 // vars/buildRpm.groovy
+/*
+ * Copyright (C) 2020-2024 Intel Corporation
+ * Copyright (C) 2025 Hewlett Packard Enterprise Development LP
+ */
```

Comment:
The (C) actually should not be in Copyright messages.   It has no legal meaning, and we should only have it in places where we have to match a known boilerplate that has it.

I have been removing  '(C)' when I update copyright lines in code modules.